samples/07.advanced-web-chat-apps/e.sso-on-behalf-of-authentication/bot/oAuthHelpers.js

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const { AttachmentLayoutTypes, CardFactory } = require('botbuilder');
+const { SimpleGraphClient } = require('./simple-graph-client');
+
+/**
+ * These methods call the Microsoft Graph API. The following OAuth scopes are used:
+ * 'OpenId' 'email' 'Mail.Send.Shared' 'Mail.Read' 'profile' 'User.Read' 'User.ReadBasic.All'
+ * for more information about scopes see:
+ * https://developer.microsoft.com/en-us/graph/docs/concepts/permissions_reference
+ */
+class OAuthHelpers {
```

Comment:
File name should be `oauthHelpers.js` or `OAuthHelpers.js`, instead of `oAuthHelpers.js`.

`OAuthHelpers.js` seems better because this is helper class.

BTW, do we need helper class or just a simple object is enough?